### PR TITLE
Issue 176

### LINKED ISSUE #176: Disease table will repeatedly re-initialize in certain cases
Currently we use the Table `item_count()` method to check if the `disease_concepts` or `disease_metadata` tables are empty, and if they are, we auto repopulate them. However, `item_count` doesn't actually auto-update -- the docs say it refreshes every 6 hours on AWS, probably the same locally? -- meaning if you rebuilt the disease table a few minutes ago and refresh the therapy table, it'll also re-rebuild diseases, too (which adds like 30 minutes). 

A few possibilities:
 * Just `scan()` the metadata table, check that the sources are listed, leave it at that
 * Do a single `scan()` on diseases to verify that it's non-empty and move on
 * Only update disease via a command line flag... or offer a command line flag to suppress updating of other resources

### PR DESCRIPTION
**Pull request by jsstevenson**
Closes #176

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +64,7 @@ export RXNORM_API_KEY={rxnorm_api_key}
 ```
 
 #### Update source(s)
-The sources we currently use are: ChEMBL, NCIt, DrugBank (CC0 data only), RxNorm, ChemIDplus, Wikidata, and HemOnc.org.
+The Therapy Normalizer currently aggregates therapy data from [ChEMBL](https://www.ebi.ac.uk/chembl/), [NCIt](https://ncithesaurus.nci.nih.gov/ncitbrowser/), [RxNorm](https://www.nlm.nih.gov/research/umls/rxnorm/index.html), [DrugBank](https://go.drugbank.com/), [ChemIDPlus](https://chem.nlm.nih.gov/chemidplus/), [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page), and [HemOnc.org](https://hemonc.org/wiki/Main_Page).
```
Comment (korikuzma) [inline README.md:67]: DrugBank, RxNorm, ChemIDplus??

**Review by korikuzma — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jsstevenson) [reply README.md:67]: Never heard of em

Comment (jsstevenson) [reply README.md:67]: (they're there, just highlighted weirdly I think?)

Comment (korikuzma) [reply README.md]: Ope you're right. My bad. That is weird!

**Pull request merged**